distributionviewer/api/views.py

```diff
@@ -32,8 +32,8 @@ def render_numeric(dist):
     return s.data
 
 
+@login_required
```

Comment:
That's a good question. As it turns out, the decorator isn't required but it does change the result. When the decorator isn't present, the API outputs the following:

```
{"metrics":{"detail":"Authentication credentials were not provided."}}
```

When the decorator is present, the user is redirected to the login page when trying to access API endpoints.

But you probably already knew that. I'm happy either way. Which do you think would be better?